test/smoke/axon-noeg/axon_agent_test.rb

```diff
@@ -0,0 +1,90 @@
+# # encoding: utf-8
+
+# Inspec test for recipe tripwire_agent::axon_agent
+
+# The Inspec reference, with examples and extensive documentation, can be
+# found at http://inspec.io/docs/reference/resources/
+
+if os.windows?
+  config_dir = 'C:\ProgramData\Tripwire\agent\config'
+  config_file = config_dir + '\twagent.conf'
+  tag_file = config_dir + '\metadata.yml'
+  reg_key_file = config_dir + '\registration_pre_shared_key.txt'
+  pkg_name = 'Axon Agent'
+  srvc_nme = 'TripwireAxonAgent'
+  eg_srvc_nme = 'TripwireEventGeneratorService'
+  eg_process_nme = 'tesvc.exe'
+else
+  config_dir = '/etc/tripwire'
+  config_file = config_dir + '/twagent.conf'
+  tag_file = config_dir + '/metadata.yml'
+  reg_key_file = config_dir + '/registration_pre_shared_key.txt'
+  pkg_name = 'axon-agent'
+  srvc_nme = 'tripwire-axon-agent'
+  eg_srvc_nme = 'tw-eg-service'
+  eg_process_nme = 'tesvc'
+end
+
+# Axon config directory should exist
+describe directory(config_dir) do
+  it { should exist }
+end
+
+# Configuration file should exist and have the bridge, port, and spool size set
+describe file(config_file) do
+  it { should exist }
+  its('content') { should match /bridge\.host=tw-bridge\.example.com/ }
+  its('content') { should match /bridge\.port=5670/ }
+  its('content') { should match /spool.size.max=1g/ }
+
+  # Following should not exist in the file
+  its('content') { should_not match /dns\.service\.name=bridge/ }
+  its('content') { should_not match /dns\.service\.domain=example\.com/ }
+  its('content') { should_not match /socks5\.*/}
+  its('content') { should_not match /tls\.version*/ }
+  its('content') { should_not match /tls\.cipher\.suites*/ }
+end
+
+# Tag file for the Axon agent should exist with properly content
```

Comment:
s/properly/proper/ ?

Reply:
Properly content, with the content you should have properly. :D thanks for the catch.